docs/whats_new/v0.1.1.md

```diff
@@ -0,0 +1,57 @@
+# Version 0.1.1
```

Comment:
Finally, do we want to call this 0.2? It isn't usual for a micro release to drop support of a Python version and add support of another. This is probably worth being v0.2.

Reply:
Plus adding TImeSeries suppport is a major feature

Reply:
Sure. I think that makes sense